service/src/main/java/uk/gov/hmcts/reform/fpl/model/order/HearingOrder.java

```diff
@@ -212,4 +212,10 @@ public List<String> getDocumentAcknowledge() {
     public boolean isConfidentialOrder() {
         return isNotEmpty(orderConfidential);
     }
+
```

Comment:
I'm guessing for this one and the GeneratedOrder methods there's going to no danger of returning a confidential order when we shouldn't

Reply:
No, I only refactored those getting the order which copying the same logic `(isConfidentialOrder()) ? orderConfidential : order;`